weblate/trans/tests/test_hooks.py

```diff
@@ -1562,15 +1643,41 @@ class AzureBackendTest(HookBackendTestCase):
     def test_ping(self):
         self.assert_hook('{"diagnostics": "ping"}', None)
 
-    def test_git(self):
+    def test_git_old(self):
         url = "https://f.visualstudio.com/c/_git/ATEST"
         self.assert_hook(
-            AZURE_PAYLOAD,
+            AZURE_PAYLOAD_OLD,
             {
                 "branch": "master",
                 "full_name": "ATEST",
                 "repo_url": url,
-                "repos": [url],
+                "repos": [
+                    "https://dev.azure.com/f/c/_git/ATEST",
+                    "https://dev.azure.com/f/be9b3917-87e6-42a4-a549-2bc06a7a878f/"
+                    "_git/278d5cd2-584d-4b63-824a-2ba458937249"
+                    "git@ssh.dev.azure.com:v3/f/c/ATEST",
```

Comment:
This looks wrong, comma is missing before the ssh URL. To make it clear that you're relying on string concatenation, I'd suggest something like:

```suggestion
                    (
                        "https://dev.azure.com/f/be9b3917-87e6-42a4-a549-2bc06a7a878f/"
                        "_git/278d5cd2-584d-4b63-824a-2ba458937249"
                    ),
                    "git@ssh.dev.azure.com:v3/f/c/ATEST",
```